content/reference/current/_index.md

```diff
@@ -13,4 +13,5 @@ cascade:
     gatlingSbtPluginVersion: "3.2.2"
     frontLineSbtPluginVersion: "1.3.2"
     frontLineGradlePluginVersion: "1.4.0"
+    externalPluginsVersion: "1.14.2"
```

Comment:
ciPluginsVersion?

Reply:
or frontLineCiPluginsVersion

Reply:
Atm, there is also the grafana datasource which shares the same version, so it's not really only "ciPlugins"

Reply:
Oh, fair enough. Aren't they always aligned with `revnumber` though?

Reply:
They are aligned atm, but they soon won't be.
They will have a separated version, not aligned with self-hosted or cloud.

Reply:
ok, then just remove the change about the go.mod file and merge!